config/config.exs

```diff
@@ -9,7 +9,7 @@ use Mix.Config
 # back to each application for organization purposes.
 import_config "../apps/*/config/config.exs"
 
-config :ewallet, root: File.cwd!()
+config :ewallet, root: File.cwd!(), version: "1.1.0"
```

Comment:
@sirn It's possible to dynamically get the version from the `Mixfile`'s `project.version` but I guess that doesn't work with distillery. Any suggestions?